recipes-extended/baseboxd-tools/baseboxd-tools/bundle-debug-info

```diff
@@ -190,6 +191,42 @@ function get_port_list() {
   log_cmd_output onlpdump -S
 }
 
+function get_mstpd_state() {
+  title "Retrieving MSTPD state"
+  OUTPUT_FILE="$TMPDIR/mstpd_state"
+  LST="Service State" log_cmd_output /bin/systemctl status mstpd
+
+  for interface in /sys/class/net/*; do
+    # skip non-bridges
+    [ -d "$interface/bridge" ] || continue
+    # check that userspace stp (=2) is active
+    [ "$(cat $interface/bridge/stp_state)" = "2" ] || continue
+    # get the interface name
+    ifname="$(basename $interface)"
+
+    LST="Bridge State" log_cmd_output /sbin/mstpctl showbridge $ifname
+    LST="Port Detail" log_cmd_output /sbin/mstpctl showportdetail $ifname
+
+    LST="MST Config ID" log_cmd_output /sbin/mstpctl showmstconfid $ifname
+    LST="MSTIs" log_cmd_output /sbin/mstpctl showmstilist $ifname
+    LST="VID-to-FID" log_cmd_output /sbin/mstpctl showvid2fid $ifname
+    LST="FID-to-MSTID" log_cmd_output /sbin/mstpctl showfid2mstid $ifname
+
+    # will always contain at least the CIST (MSTI 0)
+    mstis="$(/sbin/mstpctl showmstilist $ifname | tail -n +2)"
+
+    for msti in $mstis; do
+      LST="MSTI $msti Tree State" log_cmd_output /sbin/mstpctl showtree $ifname $msti
+
+      for port in $interface/brif/*; do
+        portname="$(basename $port)"
+	LST="MSTI $msti Port $portname State" log_cmd_output /sbin/mstpctl showtreeport $ifname $portname $msti
```

Comment:
```suggestion
        LST="MSTI $msti Port $portname State" log_cmd_output /sbin/mstpctl showtreeport $ifname $portname $msti
```
The line is indented with a tab instead of spaces.

Reply:
Fixed.